service-catalog/rustc-perf/README.md

```diff
@@ -13,9 +13,22 @@ It also uses the `shared` db of [rds-databases] and the `rustc-perf` S3 bucket.
 
 The collector (the machine that runs the benchmarks) is a dedicated physical server running at [Hetzner].
 
+## Versions
+
+These are the versions we need to keep up-to-date:
+
+- Node and Ubuntu version specified in the [Dockerfile]
+- Dependencies of [Cargo.lock] and [package-lock.json]
+- Postgres version of the `shared` database
+- GitHub Actions versions in [workflows]
```

Comment:
And also the Hetzner machine :( I think that it's Ubuntu will reach EOL in a few months.